src/main/java/frc/robot/commands/PlaceCone/Rotate.java

```diff
@@ -115,7 +117,7 @@ public void execute() {
     // System.out.printf(
     //     "X equals %.2f PID moves %.2f%n", poseEstimatorSubsystem.getAngle(), values[2]);
     // setpoint and atgoal don't work, just brute forced.
-    if (Math.abs(180 - poseEstimatorSubsystem.getAngle()) < 2) {
+    if (Math.abs(180 - poseEstimatorSubsystem.getAngle()) < 2){
```

Comment:
The fix here should be 
`MathUtil.inputModulus(180 - poseEstimatorSubsystem.getAngle(), -180, 180)`
which makes sure the output is between -180 and +180
With the code above, if the current angle is -179, the angle you check will be 359, instead of -1
So if you are almost at 180 from the wrong side, you won't terminate the execution